lib/rebaser-server/src/server/rebase.rs

```diff
@@ -55,7 +55,7 @@ pub(crate) async fn perform_rebase(
         to_rebase_workspace_snapshot_address,
         onto_workspace_snapshot.id().await
     );
-    info!("after snapshot fetch and parse: {:?}", start.elapsed());
+    debug!("after snapshot fetch and parse: {:?}", start.elapsed());
```

Comment:
Not very fond of these changes being commited on what could be a frontend only PR, but this is a nit